website/templates/components/sponsors_display.html

```diff
@@ -10,99 +10,18 @@
     <span class="partners-module__title"><h1>Nos commanditaires</h1></span>
     <div class="partners-module__container partners-module__container">
         {% for category in category_sponsors %}
-            <h2 class="partners-module__categories">{{category}}</h2>
-            <div class="partners-module__row">
-                {% for sponsor in category.sponsors%}
-                    <div class="partners-card" onclick = "location.href='{{sponsor.url}}';">
-                        <img class="partners-card__logo" src={{sponsor.image}} alt={{sponsor.title}}>
-                            <span class="partners-card__text">{{sponsor.title}}</span>
-                    </div>
-                {% endfor %}
-            </div>
+            {% if category.count > 0 %}
```

Comment:
T'aurais pas pu juste faire `if not category.sponsors` ?

Reply:
Effectivement, je peux également utiliser une cette logique là :) Faut dire que plusieurs chemins mènent à Rome :) Vous préférez que je fasse un if sur une liste ?

Reply:
ouaip

Reply:
Je vais le faire de cette façon même si je crois qu'on pourrait débattre un peu à ce sujet :)

Reply:
La raison pourquoi je préfère le `if not category.sponsor` (si ça marche, j'ai pas testé), c'est que ça implique moins de code. Donc au final, c'est plus facile à lire et maintenir (de mon point de vue). 

Je vois plus ou moins la pertinence d'ajouter des variables pour "compter" le nombre de sponsors alors qu'on peut déjà regarder si c'est vide.

Comme tu dis, il y a plusieurs chemins qui mène à Rome et bien que ta solution fonctionne, je crois qu'on peut prendre un chemin encore plus court :)

Reply:
Effectivement, je suis entièrement d'accord avec le fait que ça fonctionne et que ce soit plus court. Personnellement, j'aime aussi les méthodes rapides et je voulais simplement rendre la chose un peu plus paramétrable. `if category.sponsors` sera suffisant dans ce cas-ci. (J'ai fait le test) Je suis content que tu me donnes une raison :) Merci !